src/java.base/share/classes/java/util/SequencedMap.java

```diff
@@ -263,8 +263,15 @@ default V putLast(K k, V v) {
      *
      * @implSpec
      * The implementation in this interface returns a {@code SequencedSet} instance
-     * that behaves as follows. Its {@link SequencedSet#add add} and {@link
-     * SequencedSet#addAll addAll} methods throw {@link UnsupportedOperationException}.
+     * that behaves as follows. Its {@link SequencedSet#add add}, {@link
+     * SequencedSet#addAll addAll}, {@link SequencedSet#addFirst addFirst}, and {@link
+     * SequencedSet#addLast addLast} methods throw {@link UnsupportedOperationException}.
+     * Its {@link SequencedSet#getFirst getFirst} and {@link SequencedSet#getLast getLast}
+     * methods are implemented in terms of the {@link #firstEntry firstEntry} and {@link
+     * #lastEntry lastEntry} methods of this class, respectively. Its {@link
```

Comment:
Hello Stuart, some parts of this and other updated method docs use "this interface", "this map", "this class". Was it intentional to use "this class" in this newly added text?